lib/puppeteer-helpers.js

```diff
@@ -122,6 +122,35 @@ function getAttribute ($element, attributeName) {
   return $element.evaluate((el, name) => el.getAttribute(name), attributeName)
 }
 
+/**
+ * Snapshots the node in the accessibility tree corresponding to the given element
+ *
+ * @param {import('puppeteer').ElementHandle} $element - Puppeteer element handle
+ * @returns {Promise<import('puppeteer').SerializedAXNode>})} -
```

Comment:
@romaricpascal Have we got a couple of extra characters here?

```suggestion
 * @returns {Promise<import('puppeteer').SerializedAXNode>} -
```